client/src/admin/projects/View.vue

```diff
@@ -84,15 +91,43 @@
           <td>Member</td>
         </tr>
       </table>
+      <hr>
+      <h2>Admin Actions</h2>
+      <button class="btn btn-outline-danger" v-if="!project.deletedAt" v-on:click="confirmDeleteProject()">Delete Project</button>
+      <p v-else>No actions available</p>
```

Comment:
Just chatted w/ @conoramurphy, deleted projects shouldn't show up in the admin project listing, so it shouldn't be possible to get to this page :p

Reply:
Apologies for not specifying it on the ticket better.

Reply:
Ah boo, I just made it all pretty ;)

![screen shot 2018-04-11 at 10 25 38](https://user-images.githubusercontent.com/91799/38608427-34f6e838-3d73-11e8-81fe-e733be0b51c8.png)